packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -141,3 +141,39 @@ def test_if_we_have_capabilities(cluster):
     )
     assert r.status_code == 200
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
+
+
+def test_if_overlay_master_is_up(cluster):
+    r = cluster.get('/mesos/overlay-master/state')
+
+    assert r.status_code == 200
+
+    # Make sure the `dcos` overlay has been configured.
+    json = r.json()
```

Comment:
it's a lot better to do a whole dictionary comparison here so that we get py.tests's dictionary introspection behavior, and also validate there aren't unexpected fields.

So something like
```
assert r.json() == {
  "network": {
     "overlays": [
       ...
    ],
    "vtep_subnet": "44.128.0.0/20"
  }
}
```